src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/DependencyCollection{T}.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.DotNet.UpgradeAssistant.Steps.Packages
 {
-    internal class DependencyCollection<T> : IDependencyCollection<T>
+    public sealed class DependencyCollection<T> : IDependencyCollection<T>
```

Comment:
is there a reason to make it public?

Reply:
Yes, Otherwise I'm getting IncosistentAccesibility error

Reply:
You've got something set up wrong then - you shouldn't be depending on this class externally